### ISSUE TITLE
can't save changes to preferences (using firefox on mac)

### ISSUE THREAD

**Issue by levigutt** [philc/vimium#4264]
**Describe the bug**
I can't save changes to preferences. 

After adding customizations, or even just comments, I see no way of saving it. 

The bottom of the page says "No changes". 
Pressing `Ctrl+Enter` does not seem to do anything. 
I've tried `Cmd+Enter` too, since I am on Mac - this also does nothing. 

**To Reproduce**
1. go to manage addons [about:addons](about:addons)
2. click Preferences for vimium
3. add text to any text field
4. At this point, the bottom still says "No changes" and `Ctrl+Enter` (or `Cmd+Enter`) does nothing
5. leaving preferences
6. go back to preferences, and changes are gone

<img width="872" alt="image" src="https://github.com/philc/vimium/assets/16719598/60fcba5f-2d60-4ed0-a1d3-5fec899e633b">

**Browser and Vimium version**

Firefox: version 114.0.1
Vimium: version 1.67.6
OS: Mac OS Ventura 13.4 (M2)

**Comment by philc** [comment]
Yes, the Vimium Options page you're showing is in a bad state, so presumably there is some exception happening on this page. If anyone else comes across this, please reopen and post an console output or errors after clicking the Inspect button for the Vimium extension on about:debugging#/runtime/this-firefox.
